go.mod

```diff
@@ -3,22 +3,30 @@ module github.com/RedHatGov/bundle
 go 1.16
 
 require (
+	github.com/Shopify/logrus-bugsnag v0.0.0-20171204204709-577dee27f20d // indirect
 	github.com/blang/semver/v4 v4.0.0
+	github.com/bshuster-repo/logrus-logstash-hook v1.0.2 // indirect
 	github.com/google/uuid v1.2.0
-	github.com/openshift/oc v0.0.0-alpha.0.0.20210612233142-1077b0516d5b
+	github.com/joelanford/ignore v0.0.0-20210610194209-63d4919d8fb2
+	github.com/openshift/library-go v0.0.0-20210521084623-7392ea9b02ca
+	github.com/openshift/oc v0.0.0-alpha.0.0.20210721184532-4df50be4d929
+	github.com/operator-framework/operator-registry v0.0.0-00010101000000-000000000000
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/cobra v1.1.3
 	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
+	k8s.io/apimachinery v0.21.1
 	k8s.io/cli-runtime v0.21.1
 	k8s.io/klog v1.0.0
 	k8s.io/klog/v2 v2.8.0
+	rsc.io/letsencrypt v0.0.3 // indirect
 )
 
 replace (
 	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.8.7
 	github.com/apcera/gssapi => github.com/openshift/gssapi v0.0.0-20161010215902-5fb4217df13b
+	github.com/operator-framework/operator-registry => github.com/estroz/operator-registry v1.3.1-0.20210723182709-28179ca932d8
```

Comment:
While this PR is a WIP I'll be pinning to my operator-registry fork, which can be modified to expose the libraries we need quickly. Eventually I'll have to contribute these changes to the main repo, after which I'll update this pin and remove WIP status. The other components are ready to review otherwise.

Reply:
This is already in main, so resolving this comment.